build/helper/metadata_add_all.py

```diff
@@ -609,10 +609,11 @@ def _get_least_restrictive_codegen_method(codegen_methods):
 def _add_enum_value_python_name(enum_info, config):
     '''Add 'python_name' for all values, removing any common prefixes and suffixes'''
     for v in enum_info['values']:
+        v['user_set_python_name'] = 'python_name' in v
```

Comment:
Is this `user_set` nomenclature already used?
A more descriptive term could be `explicitly_set` (vs. implicit or expanded or derived).

Reply:
I've eliminated the key, as noted [here](https://github.com/ni/nimi-python/pull/2073#discussion_r1929892527).